static_src/stores/app_store.js

```diff
@@ -0,0 +1,47 @@
+
+import AppDispatcher from '../dispatcher';
+import BaseStore from './base_store.js';
+import cfApi from '../util/cf_api.js';
+import LoginStore from './login_store.js';
+import { appActionTypes } from '../constants.js';
+
+class AppStore extends BaseStore {
```

Comment:
A comment at the top of each of these stores to tell me what are the cases / capabilities of each.